apko.advisories.yaml

```diff
@@ -78,6 +78,23 @@ advisories:
           type: vulnerable-code-not-included-in-package
           note: Only affects Windows
 
+  - id: CVE-2023-45288
```

Comment:
fixed in https://github.com/wolfi-dev/os/commit/6996b9b9a6bdfbc305031f502491cc7b85dc5dbc